networkx/algorithms/bipartite/covering.py

```diff
@@ -0,0 +1,52 @@
+#    Copyright 2016 NetworkX developers.
+#    Copyright (C) 2004-2016 by
+#    Nishant Nikhil <nishantiam@gmail.com>
+#    All rights reserved.
+#    BSD license.
+
+""" Functions related to graph covers."""
+
+import networkx as nx
+from networkx.utils import not_implemented_for, arbitrary_element
+from networkx.algorithms.bipartite.matching import hopcroft_karp_matching
+from networkx.algorithms.covering import min_edge_cover as _min_edge_cover
+
+
+@not_implemented_for('directed')
+@not_implemented_for('multigraph')
+def min_edge_cover(G, matching_algorithm=None):
+    r"""Returns a set of edges which constitutes
+    the minimum edge cover of the graph.
+
+    A smallest edge cover can be found in polynomial time by finding
+    a maximum matching and extending it greedily so that all vertices
+    are covered.
+
+    Parameters
+    ----------
+    G : An undirected, unweighted bipartite graph.
+
+    matching_algorithm : function
+        It specifies the maximum weight matching algorithm to be used. If none,
+        ``hopcroft_karp_matching`` is used.
+
+    Returns
+    -------
+    min_cover : set
+        The covering is returned as a set, ``min_cover``.
+        It contains all the edges of minimum edge cover
+        in form of tuples.
+
+    Notes
+    -----
+    An edge cover of a graph is a set of edges such that every vertex of
+    the graph is incident to at least one edge of the set.
+    A minimum edge cover is an edge covering of smallest possible size.
+
+    Due to its implementation, the worst-case running time of this algorithm
+    is bounded by the worst-case running time of the function
+    :func:`matching_algorithm`
```

Comment:
Could you please give a brief introduction to when to use single tick, double ticks or `func` ?